rfcs/protocols/asset-transfer/corda.md

```diff
@@ -6,9 +6,19 @@
 # Asset Transfer Protocol Units in Corda Networks
 
 - RFC: 02-010
-- Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
+- Authors: Krishnasuri Narayanam, Sandeep Nishad, Venkatraman Ramakrishna, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 01-Jun-2021
+- Since: 21-Oct-2022
 
-## Summary
+## Weaver Asset Transfer implementation approach for Corda network
+Below we describe the approach for the implementation of asset transfer from one ledger to another that requires only non-validating notary.
+- Pledge transaction: Consumes an input *Asset State* and produces *Pledge State* as the output. A reference to the consumed *Asset State* is maintained as a Corda *StaticPointer* (instead of copying the entire *Asset State* object), which will be used during the reclaim of pledged asset later in the same ledger.
+  - only the party performing the pledge is required to be the signer of the trasaction (however, both the party performing the pledge and the recipient are considered to be the owner of _Pledge State_).
+<img src="../../resources/images/asset-transfer-corda-pledge.png" width=80%>
+- Claim-Remote-Asset transaction: The Corda _flow_ that implements this transaction triggers a _subFlow_ to fetch an object that represents the pledge status in the remote network (export network). This object already must have been written to the local ledger (import network) as part of a trusted data-sharing invocation between these networks. This object data is validated such that: (i) expiry time has not elapsed (ii) the party performing the claim is same as the recipient in the _Pledge State_ (iii) local network is same as the remote network specified in _Pledge State_. Moreover, the transaction _flow_ ensures that the Claim is not already done in the local network by checking for the non-existance of claim status state in local ledger. The Corda contract _command_ checks that there are two output states (one representing the claimed asset state and other representing the claim status state object in local ledger). The _contract_ also checks that the owner of the claimed asset is same as the recipient in the claim status state; and the party performing the Cliam is the required signer of the transaction. The Corda _flow_ that implements this transaction triggers a _subFlow_ to create asset object that is claimed as part of this transaction.
```

Comment:
Are the names actually hyphenated in the code? If so, it's fine. Otherwise, use the exact keywords from the source code.